Makefile

```
@@ -63,8 +59,6 @@ deps-ubuntu:
 		tesseract-ocr-eng \
 		tesseract-ocr \
 		wget
-	wget -O - https://github.com/nijel/tesseract-ocr-build/releases/download/4.0.0-1/tesseract.tar.xz | tar -xJf -
-	cp -rt /usr .tesseract/*
```

It's not acceptable to just remove this step entirely here. (And Circle-CI configuration is not the only use-case.)
```suggestion
	apt-get install -y software-properties-common && add-apt-repository ppa:alex-p/tesseract-ocr
```